File: boj/2139/soln2139.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"testing"
	"time"
)

var reader = bufio.NewReader(os.Stdin)
var writer = bufio.NewWriter(os.Stdout)

func main() {
	defer writer.Flush()

	// Input
	var day, month, year int
	for {
		fmt.Fscanln(reader, &day, &month, &year)
		if day == 0 && month == 0 && year == 0 {
			break
		}

		// Logic
		loc, _ := time.LoadLocation("Asia/Seoul")
		ans := time.Date(year, time.Month(month), day, 0, 0, 0, 0, loc).YearDay()

		// Output
		fmt.Fprintln(writer, ans)
	}
}

func test(t *testing.T) {

}

File: problem_solving/boj/2139/soln2139.py
"""
# 백준
# No. 2139
# Python 3.11.9
# by "nno0obb"
"""

from datetime import date


def main():
    # Input
    while True:
        day, month, year = map(int, input().split())
        if (day, month, year) == (0, 0, 0):
            break

        # Logic
        ans = int(date(year, month, day).strftime("%j"))

        # Output
        print(ans)


if __name__ == "__main__":
    main()
